lib/CachingOperation.ts

```diff
@@ -10,4 +12,27 @@ export class CachingOperation<LoadedValue> extends AbstractFlatOperation<LoadedV
       })
     }
   }
+  public setAsync(key: string, newValueFn: () => Promise<LoadedValue>): Promise<LoadedValue> {
```

Comment:
having an optional param inside `getAsync` and `get` methods would be more seamless